src/readers/FuzzyVariableParser.cpp

```diff
@@ -117,27 +118,58 @@ string FuzzyVariableParser::ParseIdxFileName(const string& rootPath, ByteArrayRe
 //  static
 size_t FuzzyVariableParser::ParseDataSize(VarType type, ByteArrayReader* reader)
 {
+  size_t len = 0;
+  std::string str;
+  std::regex re("\\d+");
+  std::smatch match;
+
     switch (type)
     {
     case DBL:
         return 8;
         break;
-    
+
+    case LNG:
+        return 4;
+        break;
+
     case INT:
         return 2;
         break;
 
-    case LNG:
-        return 4;
+    case BIN:
```

Comment:
@litalbarkai I added bin/chr/pck and moved lines inside the switch, this change allows  to read old census data (e.g., Argentina 1991)

Reply:
@litalbarkai does this make any sense?